packages/web-app-admin-settings/src/components/Spaces/ContextActions.vue

```diff
@@ -4,7 +4,7 @@
     <quota-modal
       v-if="quotaModalIsOpen"
       :cancel="closeQuotaModal"
-      :space="quotaModalSelectedSpace"
+      :spaces="items"
```

Comment:
spaces are users in that case, if so I think we should rename the component prop to something more generic

Reply:
jep

Reply:
i think the naming is right, items are space resources